optuna/cli.py

```diff
@@ -239,11 +235,12 @@ def _dump_table(records: List[Dict[str, Any]], header: List[str]) -> str:
 
 
 def _format_output(
-    records: Union[List[Dict[Tuple[str, str], Any]], Dict[Tuple[str, str], Any]],
-    columns: List[Tuple[str, str]],
+    records: list[dict[tuple[str, str], Any]] | dict[tuple[str, str], Any],
+    columns: list[tuple[str, str]],
     output_format: str,
     flatten: bool,
 ) -> str:
+
```

Comment:
An unnecessary blank line was inserted in your commit. Please consider removing it.
```suggestion
```